server/src/db/business.ts

```diff
@@ -0,0 +1,49 @@
+import pool from './pool'
+
+const businessNameTaken = 'business_name_key'
+const businessHandleTaken = 'business_handle_key'
+
+export async function createTable(): Promise<void> {
+  await pool.query(
+    `CREATE TABLE IF NOT EXISTS "business" (
+            businessID   SERIAL PRIMARY KEY,
+            userID       INTEGER NOT NULL, 
+            name         VARCHAR(20) NOT NULL UNIQUE,
+            handle       VARCHAR(20) NOT NULL UNIQUE,
+            email        VARCHAR(20) NOT NULL,
+            logo         BYTEA,
+            website      VARCHAR(32),
+            description  VARCHAR(100)
+        )`,
+  )
+}
+
+export async function create(
+  name: string,
+  email: string,
+  handle: string,
+  website: (string | undefined),
+  desciption: (string | undefined),
+  logo: (File | undefined),
```

Comment:
Some Typescript goodness:

```suggestion
  website?: string,
  desciption?: string,
  logo?: File,
```